src/js/pensions/components/MarriageTitle.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { getMarriageTitle } from '../helpers';
+
+export default function MarriageTitle({ id, formContext }) {
+  return (
+    <legend
+        className="schemaform-block-title"
+        id={id}
+        tabIndex="-1">
```

Comment:
Good question. I think this is copied and pasted a couple times. I should take it out.